Makefile

```diff
@@ -1,4 +1,4 @@
-IMAGE=reanahub/reana-env-jupyter
+IMAGE=reanahub/reana-env-jupyter:2.0.0
```

Comment:
In the makefile, it is better not to use 2.0.0 tag, so that Makefile would work for any commit even during development.  (And, when we are happy with the outcome, we would simply `docker tag ...` to mark the "latest" version as "2.0.0".)